test/verifier/verifer.test.ts

```diff
@@ -183,24 +183,28 @@ describe("Verifier tests", function () {
     });
 
     it("setRequests: a group should be formed by the groupID encoded in requests params", async function () {
-      const groupID = 1;
-      const groupRequest1 = { ...request, groupID };
-      const groupRequest2 = { ...request, requestId: 2, groupID };
+      const groupID =
+        BigInt(
+          ethers.keccak256(ethers.solidityPacked(["uint256", "uint256"], [request.requestId, 2])),
+        ) & BigInt("0x0FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF");
```

Comment:
Can you please make a helper function to calculate the groupID?

Reply:
I have made a helper function for calculation of the groupID and requestId that also appear in other tests.